resources/js/processes-catalogue/components/ProcessTab.vue

```diff
@@ -5,29 +5,359 @@
+import isPMQL from "../../modules/isPMQL";
+import ListMixin from "../../requests/components/ListMixin";
+import { FilterTable } from "../../components/shared";
+import PMColumnFilterPopover from "../../components/PMColumnFilterPopover/PMColumnFilterPopover.vue";
+import paginationTable from "../../components/shared/PaginationTable.vue";
+import TasksList from "../../tasks/components/TasksList.vue";
 import DefaultTab from "./DefaultTab.vue";
+
+const uniqIdsMixin = createUniqIdsMixin();
+
+Vue.component("AvatarImage", AvatarImage);
+
 export default {
-  components: { DefaultTab },
+  components: {
+    PMColumnFilterPopover,
+    paginationTable,
+    TasksList,
+    DefaultTab,
+  },
+  mixins: [uniqIdsMixin, ListMixin],
+  props: {
+    currentUser: {
+      type: Object,
+    },
+    process: {
+      type: Object,
+    },
+  },
+  data() {
+    return {
+      pmqlRequest: `(status = "In Progress") AND (requester = "${this.currentUser.username}")`,
+      pmqlTask: `(status = "In Progress")`,
+      filter: "",
+      previousFilter: "",
+      previousPmql: "",
+      orderBy: "id",
+      orderDirection: "DESC",
+      additionalParams: "",
+      advanced_filter: "",
+      sortOrder: [
+        {
+          field: "id",
+          sortField: "id",
+          direction: "desc",
+        },
+      ],
+      fields: [],
+      showTabRequests: false,
+      showTabTasks: false,
+      tableHeaders: [],
+      tableHeadersRequests: [
+        {
+          label: "REQUEST",
+          field: "name",
+          sortable: true,
+          default: true,
+          width: 110,
+          truncate: true,
+        },
+        {
+          label: "STATUS",
+          field: "status",
+          sortable: true,
+          default: true,
+          width: 150,
+        },
+        {
+          label: "DUE DATE",
+          field: "due_at",
+          format: "datetime",
+          sortable: true,
+          default: true,
+          width: 120,
+        },
+      ],
+      tableHeadersTasks: [
+        {
+          label: "NAME",
+          field: "element_name",
+          sortable: true,
+          default: true,
+          width: 140,
+          truncate: true,
+        },
+        {
+          label: "REQUEST",
+          field: "request",
+          sortable: true,
+          default: true,
+          width: 140,
+          truncate: true,
+        },
+        {
+          label: "STATUS",
+          field: "status",
+          sortable: true,
+          default: true,
+          width: 100,
+        },
+        {
+          label: "DUE DATE",
+          field: "due_at",
+          format: "datetime",
+          sortable: true,
+          default: true,
+          width: 140,
+        },
+      ],
+      dataRequests: {},
+      dataTasks: {},
+      savedSearch: false,
+      queryTask: "",
+      queryRequest: "",
+      perPage: 10,
+    };
+  },
+  computed: {
+    endpoint() {
+      if (this.savedSearch !== false) {
```

Comment:
uhmmm I think so this is not necesary in our list...

Reply:
solved. It was removed